docs/sources/installation/tanka.md

```diff
@@ -62,6 +62,11 @@ loki + promtail + gateway {
     namespace: 'loki',
     htpasswd_contents: 'loki:$apr1$H4yGiGNg$ssl5/NymaGFRUvxIV1Nyr.',
 
+    // Set to true if using boltdb shipper and uncomment boltdb_shipper_shared_store below
+    using_boltdb_shipper: false,
+
```

Comment:
@sandeepsukhani should we mention about `index_period` here. I understand it only matters if default value differs during migration. Still wondering should we still make a note? WDYT?

Reply:
I think we should rewrite this example config with boltdb shipper as index store because we are defaulting to it now.
Regarding comment, I think it should be there in source jsonnet but I don't mind it being added here too.